README.md

```diff
@@ -73,18 +73,10 @@ Thanks for taking the time to join our community and start contributing!
 ### Development environment
 
 ```bash
-mkdir -p $(go env GOPATH)/src/k8c.io
-cd $(go env GOPATH)/src/k8c.io
 git clone git@github.com:kubermatic/kubermatic
```

Comment:
General question: Git suggests git@github.com:kubermatic/kubermatic.git as the URL (note the .git extension). I know it's not needed, but I always specify it when using SSH URLs. Without them, they look .... naked to me. :grin:

Reply:
Yeah I'm fine with adding that, it did look weird to me as well and I had to double-check if it was actually a valid clone URL.